mage/query-modules/python/xml_module.md

```diff
@@ -0,0 +1,614 @@
+      <year>2022</year>
+      <publisher>Publisher A</publisher>
+    </publication>
+  </book>
+  <book id="2">
+    <title>Book 2</title>
+    <author>Author 2</author>
+    <publication>
+      <year>2023</year>
+      <publisher>Publisher B</publisher>
+    </publication>
+  </book>
+</catalog>
+```
+Cypher: 
+
+```cypher
+RETURN xml_module.parse("", true,"/home/demonstration/Documents/file.xml") AS value;
+```
+
+Output:
+
+```plaintext
+{
+   "_catalog": [
+      {
+         "_book": [
+            {
+               "_text": "Book 1",
+               "_type": "title"
+            },
+            {
+               "_text": "Author 1",
+               "_type": "author"
+            },
+            {
+               "_publication": [
+                  {
+                     "_text": "2022",
+                     "_type": "year"
+                  },
+                  {
+                     "_text": "Publisher A",
+                     "_type": "publisher"
+                  }
+               ],
+               "_type": "publication"
+            }
+         ],
+         "_type": "book",
+         "id": "1"
+      },
+      {
+         "_book": [
+            {
+               "_text": "Book 2",
+               "_type": "title"
+            },
+            {
+               "_text": "Author 2",
+               "_type": "author"
+            },
+            {
+               "_publication": [
+                  {
+                     "_text": "2023",
+                     "_type": "year"
+                  },
+                  {
+                     "_text": "Publisher B",
+                     "_type": "publisher"
+                  }
+               ],
+               "_type": "publication"
+            }
+         ],
+         "_type": "book",
+         "id": "2"
+      }
+   ],
+   "_type": "catalog"
+}
+```
+</TabItem>
+
+</Tabs>
+
+
+## Procedures
+
+### `load(xml_url, simple, path, xpath, headers)`
+
+Loads and parses an XML file from a URL or a local file. Supports simple mode, and XPath expressions.
+
+#### Input:
+
+- `xml_url: string` ➡ input URL where the XML file is located.
+- `simple: bool (default = false)` ➡ bool used for specifying whether simple mode should be used.
+- `path: string (default = "")` ➡ path to the XML file which is to be parsed. If path is not empty, xml_input string is ignored, and only the file is parsed.
+- `xpath: string (default = "")` ➡ XPath expression. If the expression is not empty, the result of the procedure is all elements satisfying the expression.
+- `headers: Map (default = {})` ➡ map of additional HTTP headers used when fetching a file from URL.
+
+#### Output:
+
+- `output_map: Map` ➡ parsed XML map.
+
+If the XPath expression is not empty, the output is all elements which satisfy the expression.
+
+#### Simple configuration explanation
+
+For details about simple configuration, go to [**Simple configuration explanation in parse function**](#simple-configuration-explanation).
```

Comment:
I think I would add this explanation to the introduction. SOmething like  -you can choose to execute the procedure in a simple and then explain what it means... so when i am already reading about procedures i know what it is + no header necessary, no repetition in each procedure